crates/ruma-events/Cargo.toml

```diff
@@ -19,6 +19,7 @@ html = ["dep:ruma-html"]
 markdown = ["dep:pulldown-cmark"]
 unstable-msc1767 = []
 unstable-msc2448 = []
+unstable-msc2545 = []
```

Comment:
You need to reexport this feature in the ruma crate too, and add it to the `__ci` feature of the ruma crate.

Reply:
> add it to the __ci feature of the ruma crate.

You should meant `__unstable-mscs` feature instead. :P

Reply:
Ah yes :sweat_smile: